apis/go.mod

```diff
@@ -70,18 +70,37 @@ require (
 	google.golang.org/protobuf v1.36.6 // indirect
 	gopkg.in/inf.v0 v0.9.1 // indirect
 	gopkg.in/yaml.v3 v3.0.1 // indirect
-	k8s.io/apiextensions-apiserver v0.31.12 // indirect
+	k8s.io/apiextensions-apiserver v0.33.2 // indirect
 	k8s.io/klog/v2 v2.130.1 // indirect
 	k8s.io/kube-openapi v0.0.0-20250902184714-7fc278399c7f // indirect
 	sigs.k8s.io/json v0.0.0-20241014173422-cfa47c3a1cc8 // indirect
 	sigs.k8s.io/randfill v1.0.0 // indirect
-	sigs.k8s.io/structured-merge-diff/v4 v4.6.0 // indirect
+	sigs.k8s.io/structured-merge-diff/v4 v4.7.0 // indirect
 	sigs.k8s.io/yaml v1.6.0 // indirect
 )
 
 replace github.com/openshift/api => github.com/openshift/api v0.0.0-20250711200046-c86d80652a9e //allow-merging
 
-// custom RabbitmqClusterSpecCore for OpenStackControlplane (v2.16.1_patches)
-replace github.com/rabbitmq/cluster-operator/v2 => github.com/openstack-k8s-operators/rabbitmq-cluster-operator/v2 v2.6.1-0.20250922171219-e224600b4030 //allow-merging
+// pin these to avoid later versions pulled by rabbitmq
+replace k8s.io/apimachinery => k8s.io/apimachinery v0.31.13 //allow-merging
+
+replace k8s.io/api => k8s.io/api v0.31.13 //allow-merging
+
+replace k8s.io/apiserver => k8s.io/apiserver v0.31.13 //allow-merging
+
+replace k8s.io/client-go => k8s.io/client-go v0.31.13 //allow-merging
+
+replace k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.31.13 //allow-merging
+
+replace k8s.io/cli-runtime => k8s.io/cli-runtime v0.31.13 //allow-merging
+
+replace k8s.io/code-generator => k8s.io/code-generator v0.31.13 //allow-merging
+
+replace k8s.io/component-base => k8s.io/component-base v0.31.13 //allow-merging
+
+// custom RabbitmqClusterSpecCore for OpenStackControlplane (v2.16.0_patches)
+replace github.com/rabbitmq/cluster-operator/v2 => github.com/openstack-k8s-operators/rabbitmq-cluster-operator/v2 v2.6.1-0.20250929174222-a0d328fa4dec //allow-merging
```

Comment:
just wondering if we can/should also bump the version of the user branch to 2.16.0, to make it more obvious that it is really that version and not 2.6.1

Reply:
we can follow up on this